### DISCUSSION TITLE
Search dynamic pages

### DISCUSSION THREAD

**Discussion by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#1412] [Q&A]
Hi there

Is it correct that dynamic pages are not indexed?
So /productdetail (a real page) is indexed but productdetail/Id/3 is not?

**Comment by meetmandeep** [comment]
Not understood. How are you creating the dynamic pages? productdetail/id/3 is the same page with query params. The module/app creating those query params is responsible for indexing it.

**Reply by Tychodewaard** [reply]
Apparently SearchBoost is needed to get content created in Plantanapp indexed. But implementing Searchboost in Vanjaro makes Vanjaro crash.
